src/artemis/tests/test_fgs_communicator.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from artemis import fgs_communicator
+from artemis.parameters import FullParameters
+
```

Comment:
Should: These tests are a great start, thank you! I think there's a few more we can add to make it more comprehensive though:
* We can patch out the communicator nexus writers by doing `@patch("fgs_communicator.NexusWriter` and assert that correct things are done on start/stop (we should probably do this even if we don't do the asserts as currently it means the tests are doing actual file writing)
* I think a test where we have a dummy plan that just starts/stops a run then having the FGSCommunicator subscribed to a RE that runs the plan and asserting that the nexus writer mocks are called correctly would be great. This is a bit more than a unit test I guess but proves the whole stack works
* A test that when the nexus writing throws an exception the RE doesn't continue and is in an error state that we can read out to send to GDA. (I know this is more functionality of Bluesky itself but by adding an explicit test for it you're saying it's important functionality that we have thought about)